test/cpp/lite_interpreter_runtime/CMakeLists.txt

```diff
@@ -6,13 +6,22 @@ set(LITE_INTERPRETER_RUNTIME_TEST_DIR
   ${TORCH_ROOT}/test/cpp/lite_interpreter_runtime/test_lite_interpreter_runtime.cpp
 )
 
+add_library(backend_with_compiler_runtime SHARED
+        ${TORCH_ROOT}/test/cpp/jit/test_backend_compiler_lib.cpp
+        ${TORCH_ROOT}/torch/csrc/jit/backends/backend_interface.cpp
+        )
+target_compile_definitions(backend_with_compiler_runtime)
+target_link_libraries(backend_with_compiler_runtime torch)
```

Comment:
Is this line needed? Does backend_with_compiler_runtime need libtorch?

Reply:
Yes there are some references of IValue, Tensor, etc. Here I assume libtorch is the lite libtorch?

Reply:
Yeah if it's with BUILD_LITE_INTERPRETER ON.

Reply:
Added PRIVATE when linking torch to backend_with_compiler_runtime